src/content/terraform/product-landing.json

```diff
@@ -1,20 +1,20 @@
 {
 	"hero": {
-		"heading": "Terraform Placeholder Duplicate Of Vault Content",
+		"heading": "Automate Infrastructure on Any Cloud",
 		"image": "https://place-hold.it/600x320"
 	},
 	"overview": {
 		"heading": "What is Terraform?",
-		"body": "Secure, store and tightly control access to tokens, passwords, certificates, encryption keys for protecting secrets and other sensitive data using a UI, CLI, or HTTP API.",
+		"body": "Terraform is an infrastructure as code tool that lets you build, change, and version infrastructure safely and efficiently. This includes low-level components like compute instances, storage, and networking; and high-level components like DNS entries and SaaS features.",
```

Comment:
The marketing style guide specifies it as no hyphens. You're technically correct that it's acting as an adjective here and should be hyphenated, but for consistency with the marketing guidance I'm going to keep it unhyphenated here. This might be worth a bigger discussion though. Good call out!